tests/unit/backend/util/test_util.py

```diff
@@ -1,6 +1,31 @@
-from imars3d.backend.util.functions import clamp_max_workers
+# package imports
+from imars3d.backend.util.functions import clamp_max_workers, to_time_str
+
+# third party imports
+import pytest
+
+# standard imports
+from datetime import datetime
 
 
 def test_clamp_max_workers():
     assert clamp_max_workers(10) >= 1
     assert clamp_max_workers(-10) >= 1
+
+
+@pytest.mark.parametrize(
+    "timestamp",
+    [
+        datetime(2022, 1, 1, 1, 1),
+        datetime(2022, 12, 12, 12, 12),
+    ],
+)
+def test_time_str(timestamp):
```

Comment:
You aren't using the parameter `timestamp` in the actual test. It'd be better to move `datetime.now()` into the list of parameters to try
```
    [
        datetime.now(),
        datetime(2022, 1, 1, 1, 1),
        datetime(2022, 12, 12, 12, 12),
    ],
```